docs/graphql/manual/api-reference/schema-metadata-api/event-triggers.rst

```diff
@@ -85,6 +85,10 @@ Args syntax
      - true
```

Comment:
can you also do this for `webhook`? `webhook` is not required (because you can supply `webhook_from_env`). I am not sure where we can say something like "one of webhook or webhook_from_env is required". Maybe at the end of description?

Reply:
done